datastore/longhorn.go

```diff
@@ -1249,6 +1283,57 @@ func (s *DataStore) ListNodesRO() ([]*longhorn.Node, error) {
 	return s.nLister.Nodes(s.namespace).List(labels.Everything())
 }
 
+func (s *DataStore) ListNodesHaveEngineImage(image string) (map[string]*longhorn.Node, error) {
```

Comment:
How about `ListNodesWithEngineImage`?
Than the below sounds nicer `ListReadyNodesWithEngineImage`

Reply:
Thank you. Fixed